templates/registration/password_change_form.html

```diff
@@ -47,7 +60,8 @@ <h1 class="h3 mb-3 font-weight-normal">Change Password</h1>
         {% endfor %}
     {% endfor %}
 
-    <button class="btn btn-lg btn-primary btn-block" type="submit">Submit</button>  
+    <button  class="bt btn btn-lg btn-primary btn-block" type="submit">Submit</button>  
```

Comment:
Button is getting blue upon hover. maybe this is due to the `btn-primary` class. Check that.